### ISSUE TITLE
There is no document formatter for 'php'-files installed.

### ISSUE THREAD

**Issue by xjoeyvo** [RiFi2k/format-html-in-php#6]
There is no document formatter for 'php'-files installed. as VSCode says when I format document

**Comment by RiFi2k** [comment]
@xjoeyvo My extension actually doesn't register itself as a PHP formatting extension because you are only allowed one formatting extension per file type. So you would basically have to choose between formatting your HTML or formatting your PHP with another extension, you couldn't have both.

My work runs before the save action because my extension formats the HTML part and not the actual PHP code, it only makes sense you would probably want to run something like PHPCBF to format your PHP code and let my extension take care of the HTML part before the PHP formatting takes place.

TLDR - Install valeryanm.vscode-phpcbf or one of the PHP CS Fixer extensions to handle your PHP formatting.

**Comment by smlombardi** [comment]
I am getting the same error message. Nothing displays in dev tools, output window or console window.  And no code is changed at all.  Not working.  Version 1.28.0-insider (1.28.0-insider), MacOS 10.13.

**Comment by RiFi2k** [comment]
@smlombardi Do you have a PHP formatting extension installed? Because if you don't then this extension can't technically run because it's waiting to go right before your actual PHP formatting extension runs.

**Comment by RiFi2k** [comment]
@smlombardi If you want my opinion, install valeryanm.vscode-phpcbf and get yourself a good PHP ruleset file. I've been using it for months perfectly. He rewrote alot of the old legacy code from  the other plugins.

**Comment by RiFi2k** [comment]
Closing this because this extension doesn't register as a PHP formatting extension so you need to bring your own for it to work correctly.

Suggest PHPCBF https://marketplace.visualstudio.com/items?itemName=ValeryanM.vscode-phpcbf
Despite the low installs it's the most up to date one that exists. I use it everyday.
